swap_meet/vendor.py

```diff
@@ -1,2 +1,67 @@
+from .item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
```

Comment:
We can refactor this into a ternary

```suggestion
        self.inventory = inventory if inventory not None else []
```